packages/core/documentation/src/components/DocumentationDisplay.vue

```diff
@@ -267,6 +268,16 @@ watch(() => props.selectedDocument, (newVal) => {
     }
   }
 }, { deep: true, immediate: true })
+
+defineExpose({
+  handleDownloadDocument,
+  handleMarkdownUiModeChange,
+  handlePublishToggle,
+  restoreOriginalDocument,
+  handleDocument,
+  setStatus,
```

Comment:
(I know it's still a draft)

When we expose these methods we will likely want to make the function names "friendlier"

e.g. handleDownloadDocument => download